messaging/filter/service.go

```diff
@@ -137,12 +148,36 @@ func (s *Service) Init(chats []*Chat) ([]*Chat, error) {
 	return allChats, nil
 }
 
+func (s *Service) Start(checkPeriod time.Duration) {
```

Comment:
Who is the caller that provides `onNewMessages` callback? Maybe we can implement this in the caller. `filter` shouldn't be as simple as possible?

Reply:
Sure, currently shhext service is passing a callback, called whenever new messages are available